### DISCUSSION TITLE
Cannot find the option of CREATE_PYTHON while compiling bindings?

### DISCUSSION THREAD

**Discussion by YANGGANG1971** [robotology/community#390] [Q&A] [answered]
While trying to compile [Yarp bindings](http://wiki.icub.org/wiki/Python_bindings#iCub_bindingsl) for python (ccmake ../), I cannot find the option of CREATE_PYTHON (see below). By hitting t to toggle advanced mode, I cannot still find the option of CAREAT_PYTHON. 

![image](https://user-images.githubusercontent.com/55526431/76277166-47296600-62cb-11ea-9240-7bae3c33e11c.png)

The related-icub software versions are below. 

> yarp version 3.1.0 
> Ubuntu 16.04
> cmake version 3.5.1 ($ CMake  --version)
> python 2.7 
> swig 3.0.8

**Comment by traversaro** [comment, marked as answer]
As documented in https://www.yarp.it/yarp_swig.html, first you need to enable the `YARP_COMPILE_BINDINGS` option, and  then you will find the `CREATE_PYTHON` option.

**Comment by YANGGANG1971** [comment]
It works. Thanks a lot.
